source/index.ts

```diff
@@ -28,23 +26,17 @@ if (process.env['NODE_ENV'] !== 'production') {
 	bot.use(generateUpdateMiddleware())
 }
 
-bot.use(async (ctx, next) => {
-	try {
-		if (next) {
-			await next()
-		}
-	} catch (error: unknown) {
-		console.log('try send error', error)
-		await ctx.reply('😣 This happens… Please try again.')
-	}
+const safe = bot.errorBoundary(async ({error, ctx}) => {
```

Comment:
I think I would prefer `const bot = new Bot<Context>(token).errorBoundary(…` more. having it named safe is somehow weird.

Also: neat that errorBoundary exists now, way more useful than `bot.catch` in my opinion.

Reply:
> neat that errorBoundary exists now

It was added in grammY 1.1 more than 2 years ago.

> I would prefer `const bot = new Bot<Context>(token).errorBoundary(…` more

This does not type-check.

Reply:
Then rename the variable `safe` to `bot` and the current bot to something like `mainBot` or `botMain` or something like that? `safe` is not telling much about what is included and `bot` is the main thing to be worked with so it should be the short default. Or is there any other community default that evolved already?